scss/book-list.scss

```diff
@@ -0,0 +1,33 @@
+body {
```

Comment:
Don't style HTML tags directly and certainly not the body, it could create inconsistencies. Maybe you can create this space with the container